server/models/rider.ts

```diff
@@ -62,10 +62,8 @@ const schema = new dynamoose.Schema({
     validate: (email) => isEmail(email as string),
   },
   accessibility: {
-    type: Array,
-    schema: [String],
+    type: String,
     required: false,
-    default: [],
   },
```

Comment:
just for some extra security, add an enum field to accessibility (like organization below)